ibis/formats/pyarrow.py

```diff
@@ -161,7 +193,23 @@ def from_ibis(cls, dtype: dt.DataType) -> pa.DataType:
             )
             return pa.map_(key_field, value_field, keys_sorted=False)
         elif dtype.is_geospatial():
-            return pa.binary()
+            # Resolve CRS
+            if dtype.srid is None:
+                crs = None
+            elif dtype.srid == 4326:
+                crs = geoarrow_types.OGC_CRS84
+            else:
+                # Warn for dropped CRS? Or geoarrow.types would need a lookup table
+                # for srid -> PROJJSON
+                crs = None
```

Comment:
Is pyproj a dependency? Is it safe to assume that an srid refers to an EPSG code?

Reply:
Looks like it comes along, when we install `pip install 'ibis-framework[duckdb, geospatial]'`. 

```sh
Installing collected packages: pytz, tzdata, typing-extensions, toolz, sqlglot, six, pygments, 
pyarrow-hotfix, parsy,  packaging, numpy, mdurl, duckdb, click, certifi, bidict, attrs, atpublic,
shapely, python-dateutil, pyproj, pyarrow, markdown-it-py, cligj, click-plugins, rich, pandas,
 fiona, ibis-framework, geopandas
```